ethereum/ewm/BREthereumEWM.c

```diff
@@ -653,6 +663,27 @@ ewmDestroy (BREthereumEWM ewm) {
     free (ewm);
 }
 
+/// MARK: - Start/Stop
+
+extern void
+ewmStart (BREthereumEWM ewm) {
+    // Start the alarm clock.
+    alarmClockStart(alarmClock);
```

Comment:
This is the process-wide alarm clock, ya? Don't think we want to be doing this, as it has repercussions on any other handler using a timeout, if I am understanding correctly.

Reply:
CORE-489